.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
-* [ ] ❗ Have you followed the guidelines in the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) list?
+* [ ] ❗ I have followed the [Contributing to DVC](https://dvc.org/doc/user-guide/contributing/core) checklist
 
-* [ ] 📖 Check this box if this PR **does not** require [documentation](https://dvc.org/doc) updates, or if it does **and** you have created a separate PR in [dvc.org](https://github.com/iterative/dvc.org) with such updates (or at least opened an issue about it in that repo). Please link below to your PR (or issue) in the [dvc.org](https://github.com/iterative/dvc.org) repo.
+* [ ] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or at least issue) in [dvc.org](https://github.com/iterative/dvc.org). If the CLI API is changed, I have updated [tab completion scripts](https://github.com/iterative/dvc/tree/master/scripts/completion).
```

Comment:
```suggestion
* [ ] 📖 If this PR requires [documentation](https://dvc.org/doc) updates, I have created a separate PR (or issue, at least) in [dvc.org](https://github.com/iterative/dvc.org) and linked it here. If the CLI API is changed, I have updated [tab completion scripts](https://github.com/iterative/dvc/tree/master/scripts/completion).
```

Reply:
I left out the "and linked it here/there" as I can't honestly imagine anyone *not* linking things, but whatever...

Reply:
People don't always link it because since they mention the core PR in the docs PR there's an automatic link that GH makes in the body of the PR. But it's not obvious or easy to find so Ii think its best to explicitly remind people to paste the link in the PR description.